runtime/src/main/java/org/corfudb/recovery/RecoveryUtils.java

```diff
@@ -53,18 +52,17 @@ static void createSmrMapIfNotExist(CorfuRuntime runtime, UUID streamId) {
         }
     }
 
+    static AddressSpaceOptions addressSpaceOptions = new AddressSpaceOptions();
```

Comment:
Should be static final and be all caps and descriptive (something like DEFAULT_OPTIONS). Also It should be a member of AddressSpaceOptions